manifests/moco-nodes.pp

```diff
@@ -342,6 +342,13 @@
     include toplevel::server::buildduty_tools
 }
 
+# mergeday
+
+node /^mergeday\d+\.srv\.releng\.(use1|usw2)\.mozilla\.com$/ {
```

Comment:
As far as I can tell from AWS console, we have a `mergeday1` in the `us-west-2` and your [warrior PR](https://github.com/mozilla-releng/releasewarrior-2.0/pull/150/files#diff-f188716043a5f41d82e523895637b90aR5) confirms that. Is there any reason we're allowing here `use1` as well? Or are we keeping this for security reasons in case us-west-2 is down and we quickly need to rely on a newly ramped up `us-east-1` new instance?

Reply:
I wrote it like this to leave room for future expansion. I can hardcode a single hostname here if you prefer.

Reply:
Nope, makes sense. Just wanted to confirm my understanding was accurrate. Let's leave it like that, it doesn't hurt.